src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -184,6 +184,24 @@ def test_get_fov(self):
         with self.assertRaises(TypeError):
             tiled_acq_task._getFov(None)
 
+    def test_compressed_stack(self):
+        """
+       Test the whole procedure (acquire compressed zstack + stitch) of acquireTiledArea function
+       """
+        # With fm streams
+        settings_obs = SettingsObserver([self.stage])
+        fm_fov = compute_camera_fov(self.ccd)
+        # Using "songbird-sim-ccd.h5" in simcam with tile max_res: (260, 348)
+        area = (0, 0, fm_fov[0] * 2, fm_fov[1] * 2)  # left, top, right, bottom
+        overlap = 0.2
+        zsteps = 21
+        future = acquireTiledArea(self.fm_streams, self.stage, area=area, overlap=overlap, settings_obs=settings_obs, zsteps=zsteps)
+        data = future.result()
+        self.assertEqual(future._state, FINISHED)
```

Comment:
Please refrain from checking internal state. This could be written `assertTrue(future.done())`